assets/_pre_prod/publish/v1.0/features/ro.txt

```diff
@@ -0,0 +1,14 @@
+Etar este un calendar open source cu design material, pentru toată lumea!
+
+Caracteristici:
+★ Vedere lunară.
+★ Vedere săptămânală, zilnică sau agendă.
+★ Folosește sincronizarea din Android. Este compatibil cu Google Calendar, Exchange, etc.
+★ Design material.
+★ Temă luminoasă și întunecată.
+★ Gratuit, open source și fără reclame.
+★ Widget cu agendă.
+★ Suportă partajarea calendarului prin ics și importarea/exportarea pe sdcard.
+
+Probleme & Feedback: https://github.com/Etar-Group/Etar-Calendar/issues 
```

Comment:
Hi!

I wanted not to say "bugs" and say "probleme" because "probleme" means problems. It is easier for people to understand than saying bugs. If you don't like that, you can modify back to bugs, it's your choice. Have a nice day!